homeassistant/components/folder_watcher.py

```diff
@@ -0,0 +1,85 @@
+"""
+Component for monitoring activity on a folder.
+"""
+import os
+import logging
+import voluptuous as vol
+from watchdog.observers import Observer
```

Comment:
Then also move these imports into local imports. For example https://github.com/home-assistant/home-assistant/blob/c462292e4d3017401a7def24ca8051b34ad605f1/homeassistant/components/media_player/cast.py#L355

Reply:
@OttoWinter If I do this:
![image](https://user-images.githubusercontent.com/11855322/37137377-5d92e8d2-229d-11e8-9623-e88c40a1787e.png)

I get the error: 
```
  File "/Users/robincole/.homeassistant/custom_components/folder_watcher.py", line 63, in <module>
    class WatcherHandler(FileSystemEventHandler):
NameError: name 'FileSystemEventHandler' is not defined
```